embedding/csharp/Tizen.Flutter.Embedding/Interop/Interop.cs

```diff
@@ -68,7 +68,7 @@ public struct FlutterDesktopEngineProperties
             FlutterDesktopEngine engine);
 
         [DllImport("flutter_tizen.so")]
-        public static extern IntPtr FlutterDesktopPluginRegistrarGetNativeWindow(
+        public static extern FlutterDesktopView FlutterDesktopPluginRegistrarGetView(
```

Comment:
You'd just remove this function and `FlutterDesktopViewGetNativeHandle` since they are unused. They can be added later when they are used again.

Reply:
There is a [plugin](https://github.com/flutter-tizen/plugins/pull/400) that requires that APIs. 
I think this PR needs to add that APIs.

Reply:
No. `Interop.cs` is only used within the C# embedding and has nothing to do with plugins.

Reply:
Then, as you mentioned, I will remove these APIs from `Interop.cs`.